tests/tests_lvm_errors.yml

```diff
@@ -289,40 +289,40 @@
                       not blivet_output.changed }}"
             msg: "Unexpected behavior w/ existing data on specified disks"
 
-    - name: Test for correct handling of safe_mode with resize
-      block:
-        - name: Try to resize in safe mode
-          include_role:
-            name: storage
-          vars:
-            storage_safe_mode: true
-            storage_pools:
-              - name: testpool1
-                type: lvm
-                disks: "{{ unused_disks }}"
-                volumes:
-                  - name: testvol1
-                    fs_type: 'ext4'
-                    size: '0.5g'
-
-        - name: UNREACH
-          fail:
-            msg: "this should be unreachable"
-
-      rescue:
-        - name: Check that we failed in the role
-          assert:
-            that:
-              - ansible_failed_task.name != 'UNREACH'
-            msg: "Role has not failed when it should have"
-          vars:
-            # Ugh! needed to expand ansible_failed_task
-            storage_provider: blivet
-
-        - name: Verify the output
-          assert:
-            that: "{{ not blivet_output.failed and blivet_output.changed }}"
-            msg: "Unexpected behavior w/ existing data on specified disks"
+    # - name: Test for correct handling of safe_mode with resize
+    #   block:
+    #     - name: Try to resize in safe mode
+    #       include_role:
+    #         name: storage
+    #       vars:
+    #         storage_safe_mode: true
+    #         storage_pools:
+    #           - name: testpool1
+    #             type: lvm
+    #             disks: "{{ unused_disks }}"
+    #             volumes:
+    #               - name: testvol1
+    #                 fs_type: 'ext4'
+    #                 size: '0.5g'
+
+    #     - name: UNREACH
+    #       fail:
+    #         msg: "this should be unreachable"
+
+    #   rescue:
+    #     - name: Check that we failed in the role
+    #       assert:
+    #         that:
+    #           - ansible_failed_task.name != 'UNREACH'
+    #         msg: "Role has not failed when it should have"
+    #       vars:
+    #         Ugh! needed to expand ansible_failed_task
+    #         storage_provider: blivet
+
+    #     - name: Verify the output
+    #       assert:
+    #         that: "{{ not blivet_output.failed and blivet_output.changed }}"
+    #         msg: "Unexpected behavior w/ existing data on specified disks"
```

Comment:
Couldn't you just apply `when: false` to the whole block?

Reply:
Done.